CHANGELOG.md

```diff
@@ -12,6 +12,25 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 * **Security**: in case of vulnerabilities.
 
 ## Unreleased
+### Added
+- Implement strums `EnumIter` on `Observer`. Allows easy iteration over all available observers.
+
+### Changed
+- Change the return type of `Observer::spectral_locus_by_index` from `[f64; 2]` to
+  `Option<[f64; 2]>`. Allows returning `None` for invalid indices.
+
+### Fixed
+- Fix caching bug in `Observer::spectral_locus_index_min` and
+  `Observer::spectral_locus_index_max`. Previously only the values for the first
+  observer it was called on was returned for all subsequent calls.
+- Fix caching bug in `RgbSpaceData::primaries_as_colorants`. Previously only the values
```

Comment:
This is the change made in #31. I forgot to add it to the changelog back then, so adding it now.